app/controllers/RestApi.scala

```diff
@@ -87,6 +101,118 @@ class RestApi @Inject() (
 
   }}
 
+  case class CreateUserBody(
+     email: String,
+     firstName : String,
+     lastName: String,
+     mobilePhone: String,
+     placeOfResidence: String,
+     birthday: Long,
+     sex: String,
+     password: String,
```

Comment:
Add possibility to send user without password.

Reply:
When we create user without an password, must we consider separate steps in the workflow? e.g. send a mail with a link, that the user can set the password? Or how will the login works in this case?

Reply:
User which are created per api never have a password and are not confirmed. 
The body will be enhanced by an flag, which defines whether the confirm mail will be send immediately or in case the user use the system the first time. 

The Workflow for the confirm mail will be defined in an separate issue.